config/packages/_sylius.yaml

```diff
@@ -29,15 +24,32 @@ sylius_theme:
                 - "%kernel.project_dir%/themes"
 
 sylius_ui:
-    events:
-        sylius.admin.login.form.content:
-            blocks:
-                test_credentials:
-                    template: 'Admin/Security/Login/testCredentials.html.twig'
-                    priority: 30
+#    events:
+#        sylius.admin.login.form.content:
+#            blocks:
+#                test_credentials:
+#                    template: 'Admin/Security/Login/testCredentials.html.twig'
+#                    priority: 30
 
 sylius_product:
     resources:
         product:
             classes:
                 model: App\Entity\Product\Product
+
+payum:
+    storages:
+        "%sylius.model.order.class%": { doctrine: orm }
+        "%sylius.model.payment.class%": { doctrine: orm }
+
+    security:
+        token_storage:
+            Sylius\Bundle\PayumBundle\Model\PaymentSecurityToken: { doctrine: orm }
+
+    dynamic_gateways:
+        config_storage:
+            Sylius\Bundle\PayumBundle\Model\GatewayConfig: { doctrine: orm }
+
+    gateways:
+        offline:
+            factory: offline
```

Comment:
Couldn't you import `- { resource: "@SyliusPayumBundle/Resources/config/app/config.yaml" }` instead of this config?